examples/async.py

```diff
@@ -5,50 +5,48 @@
 #
-# DocRaptor supports many options for output customization, the full list is
-# https://docraptor.com/documentation/api#api_general
+# DocRaptor supports many CSS and API options for output customization. Visit
+# https://docraptor.com/documentation/ for full details.
 #
 # You can run this example with: python async.py
 
 import docraptor
 import time
-import shutil
 
 doc_api = docraptor.DocApi()
-doc_api.api_client.configuration.username = 'YOUR_API_KEY_HERE' # this key works for test documents
-# doc_api.api_client.configuration.debug = True
+# this key works in test mode!
+doc_api.api_client.configuration.username = 'YOUR_API_KEY_HERE'
 
 try:
+    # different method than the synchronous documents
+    response = doc_api.create_async_doc({
+        'test': True,  # test documents are free but watermarked
+        'document_type': 'pdf',
+        'document_content': '<html><body>Hello World!</body></html>',
+        # 'document_url': 'https://docraptor.com/examples/invoice.html',
+        # 'javascript': True,
+        # 'prince_options': # {
+        #    'media': 'print', # @media 'screen' or 'print' CSS
+        #    'baseurl': 'https://yoursite.com', # the base URL for any relative URLs
+        # },
+    })
 
-  create_response = doc_api.create_async_doc({
-    "test": True,                                                   # test documents are free but watermarked
-    "document_content": "<html><body>Hello World</body></html>",    # supply content directly
-    # "document_url": "http://docraptor.com/examples/invoice.html", # or use a url
-    "name": "docraptor-python.pdf",                                 # help you find a document later
-    "document_type": "pdf",                                         # pdf or xls or xlsx
-    # "javascript": True,                                           # enable JavaScript processing
-    # "prince_options": {
-    #   "media": "screen",                                          # use screen styles instead of print styles
-    #   "baseurl": "http://hello.com",                              # pretend URL when using document_content
-    # },
-  })
-
-  while True:
-    status_response = doc_api.get_async_doc_status(create_response.status_id)
-    if status_response.status == "completed":
-      doc_response = doc_api.get_async_doc(status_response.download_id)
-      with open("/tmp/docraptor-python.pdf", "wb") as f:
-        f.write(doc_response)
-      print("Wrote PDF to /tmp/docraptor-python.pdf")
-      break
-    elif status_response.status == "failed":
-      print("FAILED")
-      print(status_response)
-      break
-    else:
-      time.sleep(1)
-
+    while True:
+        status_response = doc_api.get_async_doc_status(response.status_id)
+        if status_response.status == "completed":
+            pdf = doc_api.get_async_doc(status_response.download_id)
+            # get_async_doc() returns a binary string
+            with open("github-async.pdf", "wb") as f:
```

Comment:
Non-blocking, but why put "github" in the file name? It makes me question if this example is somehow reliant on github, though I don't think that's the case.

Reply:
Internal code generator thing that I didn't bother to remove. I'll remove in a future iteration to avoid confusion. Thanks fo noting!